Command/GenerateDtoCommand.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace WsSys\DtoGeneratorBundle\Command;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Sensio\Bundle\GeneratorBundle\Command\GeneratorCommand;
+use WsSys\DtoGeneratorBundle\Generator\XsdToDtoGenerator;
+
+/**
+ * Command to generate the Dtos
+ */
+class GenerateDtoCommand extends GeneratorCommand implements ContainerAwareInterface
+{
+    /**
+     * All the supported Types eg: xsd, json
+     * @var array 
+     */
+    protected $supportedTypes = array('xsd');
```

Comment:
The supported types are better to be define the supported types via semantic configuration for the bundle.